### PR TITLE
changed date format and wrote login tests

### PR DESCRIPTION
**Pull request by Karolis11**

### CODE REVIEW

File: roommate-app/Controllers/Login/LoginController.cs [hunk 1]
```diff
@@ -16,9 +16,10 @@ public class LoginController : ControllerBase{
     private readonly IFileCreator _file;
     ErrorLogging _errorLogging;
 
-    public LoginController(ErrorLogging errorLogging)
+    public LoginController(ErrorLogging errorLogging, IFileCreator file)
     {
-        _errorLogging = _errorLogging;
+        _errorLogging = errorLogging;
+        _file = file;
```
Comment (TisIDom) [inline roommate-app/Controllers/Login/LoginController.cs:22]: Data should be read from the database using UserService and not from a file.

File: roommate-app-testing.UnitTests/UnitTest1.cs [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+            // Assert
+            Assert.NotNull(result);
+
+            var model = result.Value as RegistrationResponse;
+            Assert.NotNull(model);
+
+            Assert.False(model.IsSuccess);
+        }
+    }
+}
```
Comment (TisIDom) [inline roommate-app-testing.UnitTests/UnitTest1.cs:141]: This whole file will have to be changed once LoginController reads from the database.

**Review by TisIDom — CHANGES_REQUESTED** [review]
All the changes needed are mentioned in the comments.

**Review by dominykase — APPROVED** [review]
Tests look good, but not sure if Login and Registration test should be in the same file. Letting it pass for now.

**Review by TisIDom — APPROVED** [review]

**Pull request merged**